test/unit/application.spec.js

```diff
@@ -65,20 +65,6 @@ describe('marionette application', function() {
     });
   });
 
-  describe('when an app has been started, and registering another initializer', function() {
```

Comment:
a clarifying question, maybe: what's the plan to support loading sub-apps at runtime, and having them start up? 

that's the primary reason i built initializers, back in the day. each sub-app had a reference to the primary app and would register itself when it was loaded, ensuring the sub-app was started either when the app starts or when the sub-app was loaded.